snet-ui/FAQ/index.tsx

```diff
@@ -0,0 +1,86 @@
+import Accordion from "@mui/material/Accordion";
+import AccordionSummary from "@mui/material/AccordionSummary";
+import AccordionDetails from "@mui/material/AccordionDetails";
+import Typography from "@mui/material/Typography";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import AddSharpIcon from "@mui/icons-material/AddSharp";
+import { styled } from "@mui/material/styles";
+import { Grid } from "@mui/material";
+import { Box } from "@mui/system";
+import Paper from "@mui/material/Paper";
+import { Button } from "@mui/material";
+//import { faqSampleData } from "sample-data/faq";
+import { faqSampleData } from "./faq";
+
+const AccordionContainer = styled(Paper)`
+  margin: 20px 20px 0 20px;
+`;
+
+// const Accordion = styled(MUIAccordion)`
+//   flex: none;
+//   transition: fill 0.25s;
+//   width: 48px;
+//   height: 48px;
+
+//   &:hover {
+//     fill: rebeccapurple;
+//   }
+// `;
+
+export default function FAQPage() {
+  return (
+    <Box>
+      <Typography align="center" color="bgtext.main" variant="h4">
+        Frequently Asked Questions
+      </Typography>
+      <Box sx={{ m: 6 }}>
+        <AccordionContainer sx={{ m: 15 }}>
+          {faqSampleData.map((faq) => (
+            <Accordion>
```

Comment:
Add key to this item.

Whenever you use map to display an array of items. You must always use a unique `key` in the listItem.